test/runner.py

```diff
@@ -504,7 +509,7 @@ def assert_tor_is_running(self):
     if process_status is None:
       return True
     else:
-      process_output = (self._tor_process.stdout.read() + '\n\n' + self._tor_process.stderr.read()).strip()
+      process_output = (self._tor_process.stdout.read().decode() + '\n\n' + self._tor_process.stderr.read().decode()).strip()
```

Comment:
I'd prefer the following for readability:

```python
def assert_tor_is_running(self):
    """
    Checks if our tor process is running. If not, this prints an error and
    provides **False**.
    """

    process_status = self._tor_process.poll()  # None if running
    if process_status is None:
        return True

    stdout = self._tor_process.stdout.read().decode()
    stderr = self._tor_process.stderr.read().decode()
    process_output = "%s\n\n%s" % (stdout, stderr)
    println('\n%s\nOur tor process ended prematurely with exit status %s\n%s\n\n%s' % ('=' * 60, process_status, '=' * 60, process_output), ERROR)
```

Reply:
agreed